MasterFramework Dev/Framework/Controls/CheckBox.lua

```diff
@@ -40,5 +40,6 @@ function framework:CheckBox(scale, action)
     end
 
     checkbox:SetChecked(checked)
+    action(checkbox, checked)
```

Comment:
I'd prefer we don't call this, and leave the caller to decide whether the action needs to be performed to set initial state or not